html2asketch/model/base.js

```diff
@@ -52,6 +58,7 @@ class Base {
       'layerListExpandedType': 0,
       'name': this._name || this._class,
       'nameIsFixed': false,
+      'resizesContent': this._resizingConstant === resizingConstraintValues.none ? 0 : 1,
```

Comment:
that's a Sketch thing, not something custom, right? I wonder why they need it 🤔

Reply:
Looks like this is something sketch adds if any of the resize controls are turned on. So Gonna remove this since it works the same with or without it.